package.json

```diff
@@ -38,22 +39,26 @@
   "devDependencies": {
     "@testing-library/react-hooks": "^5.0.2",
     "@types/jest": "^26.0.4",
+    "@types/lodash.debounce": "^4.0.6",
     "@types/lodash.get": "^4.4.6",
     "@types/lodash.set": "^4.3.6",
-    "@types/lodash.debounce": "^4.0.6",
     "@types/react": "^16.9.41",
     "@types/react-dom": "^16.9.8",
+    "eslint-config-react-app": "^7.0.1",
     "husky": "^4.3.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "tsdx": "^0.14.0",
+    "tsdx": "^0.14.1",
     "tslib": "^2.0.1",
     "typescript": "^4.0.3"
   },
   "dependencies": {
     "immer": "^9.0.6",
+    "lodash.debounce": "^4.0.8",
     "lodash.get": "^4.4.2",
-    "lodash.set": "^4.3.2",
-    "lodash.debounce": "^4.0.8"
+    "lodash.set": "^4.3.2"
+  },
+  "resolutions": {
+    "eslint": "7.17.0"
```

Comment:
See https://github.com/jaredpalmer/tsdx/issues/428. `tsdx` is a dead project, we should migrate to `tsup`.